app/.eslintrc

```diff
@@ -0,0 +1,58 @@
+{
+  "env": {
+    "amd": true,
+    "es6": true,
+    "jasmine": true,
+    "jquery": true,
+    "node": true,
+    "mocha": true,
+    "browser": true,
+    "builtin": true
+  },
```

Comment:
I think the "app/.eslintrc" can be reduced to:

```json
{
  "env": {
    "browser": true,
    "jquery": true,
    "mocha": true,
    "node": false
  },
  "rules": {
    "globals": {
      "router": false
    }
  }
}
```

With the rest of the rules being inherited from the parent .eslintrc config.